src/types/onyx/PersonalBankAccount.ts

```diff
@@ -15,6 +15,12 @@ type PersonalBankAccount = {
 
     /** Any reportID we should redirect to at the end of the flow */
     exitReportID?: string;
+
+    /** Whether the flow should continue to kyc on success */
+    shouldContinueKYCOnSuccess?: boolean;
```

Comment:
@getusha Why do we even need this value?
Can we just use `onSuccessFallbackRoute` and route if present? I don't see any specific use of this value

Reply:
Why do we even need this value?
Can we just use onSuccessFallbackRoute and route if present? I don't see any specific use of this value

Reply:
I added onSuccessFallbackRoute on top of https://github.com/Expensify/App/pull/29573/files#diff-35213b741cfc6f82c1078af63af9087af3d8de5acec7c83a3f6ef7563dd88a70R61-R63
@samh-nl do we need shouldContinueKYCOnSuccess or we can remove it?